### ISSUE TITLE
[Feature request] Open current tab in external apps

### ISSUE THREAD

**Issue by KaKi87** [kiwibrowser/android#137]
When maintaining click on a link, there is an option to open it on an external app, I'd like that option to also be available for currently open tab.

It is especially necessary when a link isn't fired by `href` but JavaScript.

**Comment by Darkshine0** [comment]
Yeah, this should be a must-have. Even for opening yt links for example, they default to the Google YouTube app and I can't choose any alternatives. But by adding this feature it would solve this problem.
